draft-ietf-quic-transport.md

```diff
@@ -664,6 +664,18 @@ packets are protected with connection- and version-specific keys
 provide confidentiality or integrity against on-path attackers, but
 provides some level of protection against off-path attackers.
 
+Servers MUST NOT send more than three times as many bytes as the number of bytes
+received prior to verifying the client's address.  Source addresses can be
+verified through an address validation token (delivered via a Retry packet or
+a NEW_TOKEN frame) or by processing any message from the client encrypted using
+the Handshake keys.  This limit exists to mitigate amplification attacks.
+
+In order to prevent this limit causing a handshake deadlock, the client SHOULD
+always send a packet upon a handshake timeout, as described in
```

Comment:
Should this be "crypto handshake timeout" now?

Reply:
How exactly is number of bytes measures?